boards/imix/src/main.rs

```diff
@@ -320,38 +303,31 @@ pub unsafe fn reset_handler() {
          &sam4l::adc::CHANNEL_AD4, // AD3
          &sam4l::adc::CHANNEL_AD5, // AD4
          &sam4l::adc::CHANNEL_AD6, // AD5
-        ],
-        192/8
-    );
+        ]);
     let adc = static_init!(
         capsules::adc::Adc<'static, sam4l::adc::Adc>,
         capsules::adc::Adc::new(&mut sam4l::adc::ADC0, adc_channels,
                                 &mut capsules::adc::ADC_BUFFER1,
                                 &mut capsules::adc::ADC_BUFFER2,
-                                &mut capsules::adc::ADC_BUFFER3),
-        864/8);
+                                &mut capsules::adc::ADC_BUFFER3));
     sam4l::adc::ADC0.set_client(adc);
 
     // # GPIO
     // set GPIO driver controlling remaining GPIO pins
     let gpio_pins = static_init!(
-        [&'static sam4l::gpio::GPIOPin; 8],
+        [&'static sam4l::gpio::GPIOPin; 7],
         [&sam4l::gpio::PC[31], // P2
          &sam4l::gpio::PC[30], // P3
          &sam4l::gpio::PC[29], // P4
          &sam4l::gpio::PC[28], // P5
          &sam4l::gpio::PC[27], // P6
          &sam4l::gpio::PC[26], // P7
          &sam4l::gpio::PA[20], // P8
-         &sam4l::gpio::PA[20], // Dummy Pin (regular GPIO)
```

Comment:
I don't know why this is still here. I _believe_ it was a hack for the rf233 user library when it was a thing. I also _believe_ it's not useful anymore. Let's remove it silently and see if anyone yells.